src/sql/workbench/browser/modal/media/modal.css

```diff
@@ -139,7 +139,8 @@
 }
 
 .modal .footer-button a.monaco-button.monaco-text-button {
-	width: 100px;
+	min-width: 100px;
+	padding: 4px 10px;
```

Comment:
Per Portal/Ibiza patterns, they have 20px padding on each side of the button: 
![image](https://user-images.githubusercontent.com/29785331/65646155-fccb5b80-dfae-11e9-9ffd-d2f789f1b0be.png)

Reply:
Thanks - looks good